composer.json

```diff
@@ -58,13 +67,16 @@
         "drupal/views_field_view": "^1.0@beta",
         "drush/drush": "^10.3",
         "islandora-rdm/islandora_fits": "dev-8.x-1.x as 1.x-dev",
-        "islandora/advanced_search": "dev-contrib",
+        "islandora/advanced_search": "^2.0.0@beta",
         "islandora/controlled_access_terms": "^2",
         "islandora/islandora": "^2.8.1",
         "islandora/openseadragon": "^2",
         "islandora/views_nested_details": "^1.0",
         "library/pdf.js": "^2.4",
-        "mjordan/islandora_workbench_integration": "^1.0"
+        "mjordan/islandora_workbench_integration": "^1.0",
+        "digitalutsc/group_solr": "^1.0.0-beta",
+        "digitalutsc/islandora_group": "^1.0.0-beta",
```

Comment:
usually the syntax is @beta (2 lines)/

Reply:
Fixed with the commit https://github.com/kylehuynh205/islandora-starter-site/commit/abc572691cebab9571b186eaf8795fa54af29361